src/execution/operator/set/physical_recursive_cte.cpp

```diff
@@ -32,8 +34,15 @@ class RecursiveCTEState : public GlobalSinkState {
 public:
 	explicit RecursiveCTEState(ClientContext &context, const PhysicalRecursiveCTE &op)
 	    : intermediate_table(context, op.GetTypes()), new_groups(STANDARD_VECTOR_SIZE) {
-		ht = make_uniq<GroupedAggregateHashTable>(context, BufferAllocator::Get(context), op.types,
-		                                          vector<LogicalType>(), vector<BoundAggregateExpression *>());
+
+		vector<BoundAggregateExpression *> payload_aggregates_ptr;
```

Comment:
nit: `vector<reference<BoundAggregateExpression>>`

Reply:
I think that would be a good idea, but AggregateHashTable needs a vector of type vector<BoundAggregateExpression *>. Therefore I've used the raw C-pointers as well. I could adapt the AggregateHashTable to use a vector<reference<...>> but not sure if that was your intention?